flyway-database-yugabytedb/src/main/java/org/flywaydb/community/database/postgresql/yugabytedb/YugabyteDBDatabaseType.java

```diff
@@ -67,4 +67,9 @@ public Parser createParser(Configuration configuration, ResourceProvider resourc
     public String getPluginVersion(Configuration config) {
         return YugabyteDBDatabaseExtension.readVersion();
     }
+
+    @Override
+    public String getDriverClass(String url, ClassLoader classLoader) {
+        return url.startsWith("jdbc:yugabytedb:") ? "com.yugabyte.Driver" : super.getDriverClass(url, classLoader);
```

Comment:
Please write a comment about why and how both url works with YugabyteDB flyway plugin. It looks like postgresql url and postgresql driver can work with YugabyteDB plugin and of course yugabytedb url will work with the YugabyteDB smart driver. Correct?

Reply:
Please remember to cover this in the documentation also.